networking/hardware_networks/about-sriov.adoc

```diff
@@ -5,12 +5,22 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can use Single Root I/O Virtualization (SR-IOV) network devices with additional networks on your {product-title} cluster for high performance applications.
+Developed by the PCI-SIG (PCI Special Interest Group), the Single Root I/O Virtualization (SR-IOV) specification is a standard for a type of PCI device assignment that can share a single device with multiple pods.
```

Comment:
Not critical, but is saying who it was developed by actually an important detail to put in the docs?